tools/peptideshaker/searchgui.xml

```diff
@@ -3,16 +3,19 @@
         Perform protein identification using various search engines and prepare results for input to Peptide Shaker
     </description>
     <macros>
-        <import>macros.xml</import>
+        <import>macros_basic.xml</import>
     </macros>
     <requirements>
 	<requirement type="package" version="@SEARCHGUI_VERSION@">searchgui</requirement>
 	<requirement type="package" version="3.0">zip</requirement>
+  <requirement type="package">JQ</requirement>
```

Comment:
indentation ... and do you mean the package `jq` here, the JSON parser? If please specify a version as well.